site/docs/components/search-field.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: Search Field
+navTitle: Search Field (not built)
+summaryParagraph: Search lets users quickly find relevant content. A search field allows a user to type a word or phrase to filter a large amount of data down to a known piece of content or to a narrower list of relevant content without navigation.
+tags: ["Search", "Searchbox", "Forms", "Text input", "Top bar", "Filters", "Autocomplete", "Autosuggest", "Live filter"]
+needToKnow:
+- All form elements including search must have a label, whether it's visually displayed or hidden.
+- We separate Autocomplete into a separate component.
+- Ensure the search input is wide enough to fit the majority of search queries informed by analytics or user research.
+- Design for loading, empty, and zero states.
+
+---
+
+import WhenToUseAndWhenNotToUse from "docs-components/WhenToUseAndWhenNotToUse"
+import WhenToUse from "docs-components/WhenToUse"
+import WhenNotToUse from "docs-components/WhenNotToUse"
+
+## Visuals
+
+### UI Kit
+
+<iframe style="border: none;" width="744" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2Fe3TyAOZKfLg9cA01Iy1Mgm%2FKaizen-Site-embed%3Fnode-id%3D19823%253A84" allowfullscreen></iframe>
```

Comment:
Not to be fixed in this PR, but this is missing "populated result" and "empty result" search states.